sql/core/src/main/scala/org/apache/spark/sql/execution/arrow/ArrowConverters.scala

```diff
@@ -52,10 +52,11 @@ private[sql] class ArrowBatchStreamWriter(
     out: OutputStream,
     timeZoneId: String,
     errorOnDuplicatedFieldNames: Boolean,
-    largeVarTypes: Boolean) {
+    largeVarTypes: Boolean,
+    largeListType: Boolean = false) {
```

Comment:
Maybe we shouldn't use the default value to avoid unexpected calls?

Reply:
I have removed it.